policyengine_us/variables/gov/states/ny/nysdps/ny_clean_heat_incentive.py

```diff
@@ -0,0 +1,74 @@
+from policyengine_us.model_api import *
+
+class ny_clean_heat_incentive(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New York State Clean Heat incentive (con Edison)"
+    documentation = "The incentive for purchasing and installing a heat pump"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://cleanheat.ny.gov/assets/pdf/CECONY%20Clean%20Heat%20Program%20Manual%206%203%2024.pdf#page=12"
+    defined_for = StateCode.NY
+
+    def formula(tax_unit, period, parameters):
+        '''
+        Calculates the incentive and the cap for the NYS Clean Heat program.
+
+        Parameters:
+        - parameters (instance)
+        - period (int)
+        - tax_unit (instance)
+
+        Returns:
+        - float: a capped incentive (float). 
+        '''
+        
+        p = parameters(period).gov.states.ny.nysdps.con_edison_clean_heat
+        project_cost = tax_unit("ny_clean_heat_project_cost", period)
+
+        family_type = tax_unit("ny_clean_heat_family_type_category", period) #var0
+        heat_pump = tax_unit("ny_clean_heat_heat_pump_category", period) #var1
+
+        # Residential
+        if family_type == 'RESIDENTIAL':
+            dac = tax_unit("ny_clean_heat_dac_category", period) #var2
+            if heat_pump == 'ASHP':
+                home = tax_unit("ny_clean_heat_home_category", period) #var3
+                heat_pump_type = tax_unit("ny_clean_heat_heat_pump_type_category", period) #var4
+                uncapped_incentive = \
+                    p.residential_amount[heat_pump][dac][home][heat_pump_type]
+            else:
+                uncapped_incentive = p.residential_amount[heat_pump][dac]
+
+            # Calculate cap
+            rate = p.rate[family_type][dac]
+            cap = project_cost * rate
```

Comment:
In the test, `rate = p.rate[family_type][dac]` using breakdown is not recognized as a float and this multiplication failed (When I changed it to a form that does not use breakdown, it was recognized.) I also tried` rate=float(rate) `but it didn't work. How can I fix this?